src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -168,5 +168,10 @@ export default {
 	button {
 		margin: 4px auto;
 	}
+
+	h2, p {
+		color: var(--color-primary-text);
```

Comment:
You want white, not primary text, otherwise you have black text on dark grey background if your primary theming color is bright

Reply:
should I hardcode white here ? then maybe I need to hardcode the background color as well instead of `--color-loading-dark`, if we agree that this is not something we want to theme.

as far as I can see here, only primary is defined: https://github.com/nextcloud/nextcloud-vue/blob/master/styleguide/assets/variables.css but if one would want to invert the colors they'd need to override "color-background-dark" with a light color ? or do we need a new `primary-text-background` color ?

Reply:
We dont want the screen to be blue or something so we should use anything related to primary.
Also we dont want it to flip ot white with dark mode, so mostlikely the loading color is wrong too.

But can be tested easily, just search for "accessibility" in your dev nextcloud, turn on dark mode, search for theming and use "#ffffff" as primary color and then start a call.

Expected result is a dark screen with readable font color

Reply:
https://github.com/nextcloud/server/blob/master/apps/accessibility/css/dark.scss#L16
Dark mode makes loading bright

Reply:
it seems that whatever combination of variables I pick, it cannot look good in both dark mode and regular mode

so if we anyway don't want to invert the colors I'll hardcode white and dark grey in this view

Reply:
=> https://github.com/nextcloud/spreed/pull/4416